adm/templates/plugins/static/cookiecutter.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "name_of_your_plugin",
+    "version": "0.0.1",
+    "one_line_summary": "one line summary",
+    "crontab_support": ["no"],
+    "license": ["Proprietary", "BSD", "MIT", "GPL", "Specific"],
+    "url": "http://yourpluginhomepage",
+    "maintainer": "Firstname FAMILYNAME <email>",
+    "vendor": "MetWork",
+    "type": ["static"]
```

Comment:
dont ask for type because there is only one valid reply !

Reply:
bootstrap_plugin.py requires cookiecutter attribute type to be defined.
That's why I'm asking for it...